dmutils/status.py

```diff
@@ -26,6 +28,19 @@ def get_flags(current_app):
     return flags
 
 
+def get_disk_space_status(low_disk_threshold=5):
+    """Accepts a single parameter that indicates the minimum percentage of disk space which should be free for the
+    instance to be considered healthy.
+
+    Returns a tuple containing two items: a status (OK or LOW) indicating whether the disk space remaining on the
+    instance is below the threshold and the integer percentage remaining disk space."""
+    disk_usage = shutil.disk_usage('/')
+
+    disk_free_percent = math.floor((disk_usage.free / disk_usage.total) * 100)
+
+    return 'OK' if disk_free_percent >= low_disk_threshold else 'LOW', disk_free_percent
```

Comment:
Not quite sure I understand how "LOW" will be used - either a healthcheck passes or it fails? Can reserve judgment until I've looked at a PR that brings this into an app :-)

Reply:
I imagine the healthcheck endpoints basically just seeing if this returns 'OK', and if not, failing the healthcheck.

Reply:
Oh I see, LOW means fail, duh :)